compiler/rustc_privacy/src/lib.rs

```diff
@@ -2080,7 +2079,7 @@ fn check_private_in_public(tcx: TyCtxt<'_>, (): ()) {
     };
     tcx.hir().walk_toplevel_module(&mut visitor);
 
-    let mut old_error_set_ancestry = HirIdSet::default();
+    let mut old_error_set_ancestry = FxIndexSet::<HirId>::default();
```

Comment:
I wonder if we should just change `HirIdSet` to be a `FxIndexSet<HirId>`

Reply:
Yeah, I considered it but I wasn't sure of the effects on other code. I'll give it a try!